.github/ISSUE_TEMPLATE/consulting.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Consulting
```

Comment:
How about a **Support request** instead, similarly to what [k8s](https://github.com/kubernetes/kubernetes/issues/new/choose) has? Personally, I like this wording better.